webplugin/js/app/mck-sidebox-1.0.js

```diff
@@ -8795,72 +8795,69 @@ var KM_ATTACHMENT_V2_SUPPORTED_MIME_TYPES = ["application","text","image"];
                     $file_remove.attr("disabled", true);
                     $mck_file_upload.attr("disabled", true);
                     KommunicateUI.hideFileBox(file, $file_box, $mck_file_upload);
-                    if (params.name === $applozic(".mck-file-box." + randomId + " .mck-file-lb a").html()) {
```

Comment:
@shubhamtibra Any specific reason to remove this check?
```
if (params.name === $applozic(".mck-file-box." + randomId + " .mck-file-lb a").html()) {
```

Reply:
This check is to stop file upload with HTML in their them right?

I thought since we're not rendering the HTML, we can allow upload.

Reply:
From my knowledge, I guess we match params.name with the text present inside the element.
We're fetching the element using jquery.

Reply:
Yeah, the upload for files like `<h1> test <h1>.txt` doesn't work with this check.